### DISCUSSION TITLE
GitLab storage

### DISCUSSION THREAD

**Discussion by cbeams** [requarks/wiki#2923] [Help / Questions] [answered]
https://docs.requarks.io/storage/git#gitlab says that GitLab support is "Coming soon!", but https://github.com/Requarks/wiki/issues/963#issuecomment-522083935 says that "any Git repository can be used". Is the documentation out of date, or am I missing something? I have not actually tried to use GitLab as a storage backend, I am just evaluating what's possible with Wiki.js at this point.

**Comment by NGPixel** [comment, marked as answer]
Any git repository can be used, including GitLab. The "Coming soon" simply refer to the documentation, not the actual capability.
